content/docs/12.5-palette-cli/10-install-palette-cli.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Install"
+metaTitle: "Install"
+metaDescription: "Learn how to install the Palette CLI and how you can use the CLI with Palette Dev Engine."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+
+
+# Installation
+
+Use the following steps to install and set up the Palette CLI.
+
+
+## Prerequisites
+
+- A Palette account. Click [here](https://console.spectrocloud.com/) to create a Palette account.
+
+
+- A Palette API key. Refer to the [API Key](/user-management/user-authentication/#apikey) reference page to learn how to create an API key.
+
+
+
+## Download and Setup
+
+1. Visit the [Downloads](/spectro-downloads#palettecli) page and download the Palette CLI by using the URL provided.
+
+
+2. Open up a local terminal session on your local system.
+
+
+3. Navigate to your defaults download folder. For Mac and Linux environments the default location is **~/Downloads** for Windows it's **C:\Users\YourUserNameHere\Downloads**. Replace `YourUserNameHere` with the name of the user account.
+
+
+4. Move the binary to a folder that is part of your system's `PATH` environment variable.
+
+
+<Tabs>
+<Tabs.TabPane tab="MacOS/Linux" key="linux-mac">
+
+Move the binary to the **/usr/local/bin** folder. Replace the download file with the name of your download file.
```

Comment:
The second sentence is awkward to me. Are we replacing an old file name with a new file name? Or are filling in a placeholder value?